app/controllers/nonsipp/landorpropertydisposal/LandOrPropertyDisposalListController.scala

```diff
@@ -54,12 +55,11 @@ class LandOrPropertyDisposalListController @Inject()(
 
   def onPageLoad(srn: Srn, page: Int, mode: Mode): Action[AnyContent] = identifyAndRequireData(srn) {
     implicit request =>
-      getDisposals(srn).map { disposals =>
-        val numberOfDisposal = disposals.map { case (_, disposalIndexes) => disposalIndexes.size }.sum
-        if (numberOfDisposal == 0) {
-          Redirect(routes.LandOrPropertyDisposalController.onPageLoad(srn, NormalMode))
-        } else {
+      val (status, incompleteDisposalUrl) = getDisposalsTaskListStatusWithLink(request.userAnswers, srn)
```

Comment:
This didn't work for me when I was doing a disposal the second time for the first land or property. It didn't detect the incomplete disposal